app/views/admin/collections/index.html.erb

```diff
@@ -13,72 +13,76 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
+
+<% unless Avalon::ControlledVocabulary.vocabulary[:units] %>
+  <% raise Avalon::VocabularyNotFound.new "Units vocabulary not found." %>
+<% end %>
```

Comment:
Is this test and raise necessary here?  How about putting this check in the `new.html.erb` view where it renders an error message (e.g. "Units vocabulary is needed for creating collections, but it is missing.") instead of the form?  That way the user can see what's wrong instead of getting a 500 or does this error get rescued higher up in an elegant way?

Reply:
For reference, that error is rescued in the [controller](https://github.com/avalonmediasystem/avalon/blob/53a77352413c21ffbc72b8bce86e60988043839c/app/controllers/admin/collections_controller.rb#L261).

Reply:
I'm not sure I know how to trigger this error in the user interface. Do you know how I can do that?

Reply:
Looking at the tests, it seems missing controlled vocabulary should redirect you to root and show a flash error message. So, it seems it's getting handled well in the code.

Reply:
:+1: , that sounds fine.
It would be nice to rework this at some point but that is totally outside the scope of this PR.